src/privatesend-client.cpp

```diff
@@ -297,25 +335,67 @@ std::string CPrivateSendClient::GetStatus()
     }
 }
 
-bool CPrivateSendClient::GetMixingMasternodeInfo(masternode_info_t& mnInfoRet)
+std::string CPrivateSendClientManager::GetStatuses()
+{
+    std::string strStatus;
+    bool fWaitForBlock = WaitForAnotherBlock();
+
+    for (auto& session : vecSessions) {
+        strStatus += session.GetStatus(fWaitForBlock) + "; ";
+    }
+    return strStatus;
+}
+
+std::string CPrivateSendClientManager::GetSessionDenoms()
+{
+    std::string strSessionDenoms;
+
+    for (auto& session : vecSessions) {
+        strSessionDenoms += (session.nSessionDenom ? CPrivateSend::GetDenominationsToString(session.nSessionDenom) : "N/A") + "; ";
+    }
+    return strSessionDenoms.empty() ? "N/A" : strSessionDenoms;
```

Comment:
Not sure I understand what you mean tbh. Any suggestions how to modify this?

Reply:
I'm not sure if `"N/A" ` is a good message to use. It seems very non-descriptive but since I don't really know what the overall use case is I can't say if it should be changed or if it's fine and dandy